search/search.py

```diff
@@ -0,0 +1,18 @@
+from flask import Blueprint, Response
+
+from lxml import etree
+
+LIBERO_NAMESPACE = 'http://libero.pub'
```

Comment:
should this use `http` or `https`? spec says `http` but thought I would double check.

Reply:
These are not URLs that will actually be loaded, they only serve the purpose to avoid clashes between different organizations that could choose the same namespace:
https://en.wikipedia.org/wiki/XML_namespace#Namespace_names